### ISSUE TITLE
rgbd: test Rgbd_Normals.compute fails on Aarch64

### ISSUE THREAD

**Issue by tomoaki0705** [opencv/opencv_contrib#1875]
##### System information (version)

- OpenCV => recent 3.4 branches
- Operating System / Platform => Ubuntu 16.04 Aarch64 (Jetson TX2)
- Compiler => GCC 5.4.0

##### Detailed description

When I ran the test ```opencv_test_rgbd``` the test fails with following message

```
$ ./bin/opencv_test_rgbd --gtest_filter=Rgbd_Normals*
CTEST_FULL_OUTPUT
OpenCV version: 3.4.3-dev
OpenCV VCS version: 3.4.3-430-g76b6f6b
Build type: Release
Compiler: /usr/bin/c++  (ver 5.4.0)
Parallel framework: pthreads
CPU features: N/A
OpenCL is disabled
Note: Google Test filter = Rgbd_Normals*
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from Rgbd_Normals
[ RUN      ] Rgbd_Normals.compute

*** FALS
* float
1 plane - input 3d points
Average error: 0.00164533 Speed: 78.0656 ms
Average error: 0.00203324 Speed: 40.9324 ms
Average error: 0.00275777 Speed: 41.806 ms
Average error: 0.00341473 Speed: 41.7944 ms
Average error: 0.00233243 Speed: 41.3095 ms
mean diff: 0.0024367
3 planes
Average error: 0.0198173 Speed: 41.1905 ms
Average error: 0.0159316 Speed: 41.0397 ms
Average error: 0.0165225 Speed: 41.4305 ms
Average error: 0.0164839 Speed: 41.4648 ms
Average error: 0.0184038 Speed: 41.316 ms
mean diff: 0.0174318
* double
1 plane - input 3d points
Average error: 0.000172128 Speed: 112.647 ms
Average error: 4.51697e-06 Speed: 70.5576 ms
Average error: 0 Speed: 63.6497 ms
Average error: 7.73532e-05 Speed: 63.5652 ms
Average error: 0.00020376 Speed: 63.3506 ms
mean diff: 9.15516e-05
/home/nvidia/opencv_contrib/modules/rgbd/test/test_normal.cpp:265: Failure
Expected: (err_mean/5) <= (errors[j][0]), actual: 9.15516e-05 vs 8e-05
 thresh: 7.9999997979030013e-05

3 planes
Average error: 0.0112171 Speed: 63.3911 ms
Average error: 0.0115122 Speed: 63.0411 ms
Average error: 0.0148569 Speed: 63.5801 ms
Average error: 0.0134869 Speed: 63.3579 ms
Average error: 0.0141238 Speed: 63.5 ms
mean diff: 0.0130394

*** LINEMOD
* float
1 plane - input 3d points
Average error: 0.00784564 Speed: 69.7648 ms
Average error: 0.00122832 Speed: 48.1081 ms
Average error: 0.000966867 Speed: 47.7332 ms
Average error: 0.000561637 Speed: 47.6521 ms
Average error: 3.65935e-05 Speed: 47.74 ms
mean diff: 0.00212781
3 planes
Average error: 0.0300315 Speed: 47.9455 ms
Average error: 0.027687 Speed: 48.0934 ms
Average error: 0.0370795 Speed: 47.9794 ms
Average error: 0.0247394 Speed: 47.8488 ms
Average error: 0.0272011 Speed: 48.0702 ms
mean diff: 0.0293477
** depth 16U - 1 plane
Average error: 0.00942854 Speed: 45.4202 ms
Average error: 0.00940987 Speed: 36.3393 ms
Average error: 0.00304545 Speed: 41.596 ms
Average error: 0.00430534 Speed: 40.7936 ms
Average error: 0.0148873 Speed: 34.7631 ms
mean diff: 0.0082153
** depth 16U - 3 plane
Average error: 0.00728355 Speed: 38.0882 ms
Average error: 0.00515288 Speed: 37.8391 ms
Average error: 0.00598059 Speed: 38.1952 ms
Average error: 0.0285222 Speed: 35.1117 ms
Average error: 0.00763396 Speed: 36.1182 ms
mean diff: 0.0109146
* double
1 plane - input 3d points
Average error: 0.00149377 Speed: 65.8469 ms
Average error: 0.00132702 Speed: 56.3374 ms
Average error: 0.000995652 Speed: 56.3908 ms
Average error: 5.19805e-05 Speed: 56.3997 ms
Average error: 0.000292746 Speed: 56.6849 ms
mean diff: 0.000832235
3 planes
Average error: 0.0405434 Speed: 70.3029 ms
Average error: 0.0259901 Speed: 58.4149 ms
Average error: 0.0286402 Speed: 54.5363 ms
Average error: 0.0338004 Speed: 54.5434 ms
Average error: 0.0266661 Speed: 54.7305 ms
mean diff: 0.031128
** depth 16U - 1 plane
Average error: 0.0186166 Speed: 60.921 ms
Average error: 0.00896744 Speed: 40.7857 ms
Average error: 0.0171228 Speed: 42.612 ms
Average error: 0.00727721 Speed: 50.7709 ms
Average error: 0.0163146 Speed: 48.6324 ms
mean diff: 0.0136597
** depth 16U - 3 plane
Average error: 0.00888076 Speed: 48.2548 ms
Average error: 0.014211 Speed: 49.9119 ms
Average error: 0.00592636 Speed: 43.5989 ms
Average error: 0.014139 Speed: 38.4047 ms
Average error: 0.003004 Speed: 42.3926 ms
mean diff: 0.00923222

*** SRI
* float
1 plane - input 3d points
Average error: 0.0185675 Speed: 89.7789 ms
Average error: 0.0150468 Speed: 76.2547 ms
Average error: 0.0178512 Speed: 62.7379 ms
Average error: 0.0185793 Speed: 67.0723 ms
Average error: 0.0115171 Speed: 65.6111 ms
mean diff: 0.0163124
3 planes
Average error: 0.0331499 Speed: 69.656 ms
Average error: 0.0319582 Speed: 65.1375 ms
Average error: 0.0332257 Speed: 67.391 ms
Average error: 0.0289857 Speed: 71.4625 ms
Average error: 0.0319995 Speed: 68.5523 ms
mean diff: 0.0318638
* double
1 plane - input 3d points
Average error: 0.0169067 Speed: 140.338 ms
Average error: 0.016083 Speed: 102.512 ms
Average error: 0.0102555 Speed: 97.8122 ms
Average error: 0.0121676 Speed: 101.264 ms
Average error: 0.0175018 Speed: 103.923 ms
mean diff: 0.0145829
3 planes
Average error: 0.0347109 Speed: 97.7507 ms
Average error: 0.0327707 Speed: 100.556 ms
Average error: 0.0350679 Speed: 95.7941 ms
Average error: 0.0313898 Speed: 92.3633 ms
Average error: 0.0320784 Speed: 99.7754 ms
mean diff: 0.0332035
[  FAILED  ] Rgbd_Normals.compute (32265 ms)
[----------] 1 test from Rgbd_Normals (32265 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (32265 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] Rgbd_Normals.compute

 1 FAILED TEST
```

Now, the key point is here 
```
mean diff: 9.15516e-05
/home/nvidia/opencv_contrib/modules/rgbd/test/test_normal.cpp:265: Failure
Expected: (err_mean/5) <= (errors[j][0]), actual: 9.15516e-05 vs 8e-05
 thresh: 7.9999997979030013e-05
```

So the error is above threshold by ```1.2e-5``` which should be a rounding error.
Loosening the eps from ```8e-5``` to ```1e-4``` should be reasonable.

I'll sned a PR later

**Comment by tomoaki0705** [comment]
I also realized that RGBD_Odometry_Rgbd.algorithmic fails on Arm/Aarch64.
I updated #1876 to fix both test failure

```
Note: Google Test filter = RGBD_Odometry_Rgbd*
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from RGBD_Odometry_Rgbd
[ RUN      ] RGBD_Odometry_Rgbd.algorithmic
/home/linaro/opencv-fork/modules/ts/src/ts.cpp:567: Failure
Failed

        failure reason: Bad accuracy
        test case #-1
        seed: ffffffffffffffff
-----------------------------------
        LOG:

Incorrect count of accurate poses [2nd case]: 90.000000 / 94.000000
-----------------------------------

[  FAILED  ] RGBD_Odometry_Rgbd.algorithmic (33626 ms)
[----------] 1 test from RGBD_Odometry_Rgbd (33626 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (33626 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] RGBD_Odometry_Rgbd.algorithmic

 1 FAILED TEST
```

**Comment by tomoaki0705** [comment]
solved by #1876
